src/Symfony/Component/HttpFoundation/UrlHelper.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation;
+
+use Symfony\Component\Routing\RequestContext;
+
+final class UrlHelper
```

Comment:
`CurrentRequestUri`?

Reply:
`CurrentRequestUri` does not sound like a service